tests/integrations/python/tests/test_anthropic.py

```
@@ -2810,6 +2811,67 @@ def test_49_web_search_sources_validation(self, anthropic_client, test_config, p
+
+            poll = anthropic_client.messages.create(
+                **request_params,
+                extra_headers={"x-bf-async-id": initial.id},
+            )
+
+            if poll.content and len(poll.content) > 0:
+                print("  Status: completed")
+                assert poll.content[0].type == "text"
+                assert len(poll.content[0].text) > 0
+                print(f"  Result: {poll.content[0].text[:80]}...")
+                print(f"✓ Async messages test passed!")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)